sensors/omron_d6t/omron_d6t.py

```diff
@@ -0,0 +1,96 @@
+import time, crcmod.predefined
+from sensors.sensor import Sensor
+
+class OmronD6T(Sensor):
+    __START_COMMAND = 0x4c
+    __ARRAY_SIZE = 16
+    __BUFFER_LENGTH = 35
+    __PEC_BUFFER = [0x15]
+    __MAX_CELLS_ABOVE_ROOM_TEMPERATURE = 10
+
+    def __init__(self, bus = 1, address = 0x0a):
+        super().__init__(bus, address, self.__START_COMMAND)
+
+    @property
+    def buffer_length(self):
+        return self.__BUFFER_LENGTH
+
+    @property
+    def occupied_status(self):
+        room_temperature, temperature_matrix = self.__read()
+
+        return self.__temperature_cells_above_threshold(room_temperature, temperature_matrix) >= self.__MAX_CELLS_ABOVE_ROOM_TEMPERATURE
```

Comment:
This is where the magic happens? Cool

Reply:
Yup. Sort of a ghetto way of determining if someone is sitting in front of the sensor but made it as tweakable as possible

Reply:
I like how it's nice and localized!